### PR TITLE
First stage of manual verification is now automatic

### PR DESCRIPTION
**Pull request by macdiesel**
@doctoryes PTAL.

There will be conflicts with your other PR. Please review this, we will merge your PR first then I will rebase and pull this in.

### CODE REVIEW

File: edxpipelines/pipelines/manual_verification.py [hunk 1]
```diff
@@ -77,9 +77,31 @@ def install_pipeline(save_config_locally, dry_run, variable_files, cmd_line_vars
...
+    # the git materials will be carried over from the first pipeline.
+    #
+    # The second stage in this pipeline requires manual approval.
+    #
+    # This allows the overall workflow to remain paused while manual verification is completed and allows the git
+    # materials to stay pinned.
+    #
+    # Once the second phase is approved, the workflow will continue and pipelines downstream will continue to execute
+    # with the same pinned materials from the upstream pipeline.
+    initial_verfication_sage = pipeline.ensure_stage(constants.INITIAL_VERIFICATION_STAGE_NAME)
```
Comment (doctoryes) [inline edxpipelines/pipelines/manual_verification.py]: Nit: `sage` -> `stage`

File: edxpipelines/pipelines/manual_verification.py [hunk 2]
```diff
@@ -77,9 +77,31 @@ def install_pipeline(save_config_locally, dry_run, variable_files, cmd_line_vars
...
+    initial_verfication_sage.set_has_manual_approval()
```
Comment (doctoryes) [inline edxpipelines/pipelines/manual_verification.py]: I thought the initial stage needed to be automatic, followed by a second stage that was manually started?

**Comment by macdiesel** [comment]
@doctoryes Done and done.

File: edxpipelines/pipelines/manual_verification.py [hunk 3]
```diff
@@ -77,9 +77,31 @@ def install_pipeline(save_config_locally, dry_run, variable_files, cmd_line_vars
...
+        ExecTask(
+            [
+                '/bin/bash',
+                '-c',
+                'echo Initial Verification run number $GO_PIPELINE_COUNTER started by $GO_TRIGGER_USER'
+            ],
+        )
+    )
 
+    manual_verification_stage = pipeline.ensure_stage(constants.MANUAL_VERIFICATION_STAGE_NAME,)
```
Comment (doctoryes) [inline edxpipelines/pipelines/manual_verification.py]: Extra comma at end.

**Comment by macdiesel** [comment]
d'oh forgot that we don't run the test suite against this repo. =(

**Pull request merged**